tests/conftest.py

```diff
@@ -63,5 +70,16 @@ def axial_slices():
     ]
 
 
+@pytest.fixture
+def axial_rgb_slices():
+    # SamplesPerPixel = 3
```

Comment:
The purpose and meaning of this comment wasn't entirely clear to me

Reply:
Yep, unnecessary - removed